### PR TITLE
Setup monasca-agent

### PR DESCRIPTION
**Pull request by toabctl**
Co-Authored-By: Sumit Jamgade <sjamgade@suse.com>

### CODE REVIEW

File: scripts/openstack-quickstart-demosetup [hunk 1]
```diff
@@ -1055,6 +1055,16 @@ EOF
         ansible-playbook -i monasca-hosts monasca.yml
     )
 
+    # do the agent setup
+    monasca-setup \
+        -u "admin" \
+        -p "$pw" \
+        --skip_enable \
```
Comment (dirkmueller) [inline scripts/openstack-quickstart-demosetup]: why?

Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: Hm. Not Sure. I used the commit from @sjamgade . But we enable the service manually later. So it should be fine.

Comment (sjamgade) [inline scripts/openstack-quickstart-demosetup]: Yes we dont want to enable it here.  Its explicitly enabled in barclamp

**Comment by sjamgade** [comment]
There is stuff missing, https://github.com/SUSE-Cloud/openstack-quickstart/pull/88 was last iirc.

--project_id is definitely needed, and the commit is lost.

File: scripts/openstack-quickstart-demosetup [hunk 2]
```diff
@@ -1055,6 +1055,16 @@ EOF
...
+        --user "monasca-agent" \
```
Comment (jgrassler) [inline scripts/openstack-quickstart-demosetup]: Sumit and me just dug through our local working copies of openstack-quickstart and there are two lines missing here (they must have gotten dropped during all that multiple-pull-requests-with-various-rebases craziness):

```
--log_dir /var/log/monasca-agent/ \
--project_name admin \
```

Without these monasca-agent is unlikely to run/work.

Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: Ah. thanks. found out the --log_dir the hard way:) Adding both now

Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: Done

**Pull request merged**